edge-modules/api-proxy-module/readme.md

```diff
@@ -111,7 +111,7 @@ For easiness of use, the API proxy module comes with a default configuration tha
 
 | Config variable  | comments |
 | ------------- |  ------------- |
-| NGINX_DEFAULT_PORT  | Changes the port Nginx listens on. If you update this environment variable, make sure the port you select is also exposed in the module dockerfile and the port binding. Default is 443.  |
+| NGINX_DEFAULT_PORT  | Changes the port Nginx listens on. If you update this environment variable, make sure the port you select is also exposed in the module dockerfile and the port binding. Default is 8000.  |
```

Comment:
Why are we changing the default? Default should be 443... that is the main reason we need the API Proxy.

Reply:
This is fine, this is the container port. This is what I did:
![image](https://user-images.githubusercontent.com/33438817/126878772-35031d7b-6eef-4dea-835a-9f3eef098527.png)

The reason for the change is. we are not root anymore, so we can't listen to port below 1024. See https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html

Reply:
So, just to make sure I understand:
We are mapping container port 8000 to 443 host port? This will require documentation updates? Also, isn't 8000 just default port (I am okay with this, just want to double check we don't have to do any other configuration for nginx)

Reply:
yes, the does documentation is updated here. It will need to be updated in the official doc too.
We don't have a choice, non-root can't listen to less than 1024.

Reply:
Yeah, makes sense.